src/App.tsx

```diff
@@ -1,39 +1,89 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
+import {
+  Button,
+  Paper,
+  Box,
+} from '@mui/material';
+import { Clock } from './Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+interface State {
+  hasClock: boolean;
```

Comment:
better wiil be name with "is"
Like 
```suggestion
  isClockVisible: boolean;
```